### PR TITLE
Added documentation for new feature inputValue as Promise 

### LINKED ISSUE #17: Add documentation for inputValue as a Promise
`inputValue` now could be a Promise (https://github.com/sweetalert2/sweetalert2/pull/1025)

Related release: https://github.com/sweetalert2/sweetalert2/releases/tag/v7.17.0

@gverni said that he will submit the PR with the documentation

### PR DESCRIPTION
**Pull request by gverni**
- Added documentation for inputValue as Promise
- Added example 

Closes #17

### CODE REVIEW

**Comment by gverni** [comment]
@limonte i added the example in the first section of the page and not in the "INPUT TYPES" section. Let me know if you'd rather want it added to the latter

File: index.html [hunk 1]
```diff
@@ -751,8 +783,8 @@ <h3 id="configuration">Configuration</h3>
       </tr>
       <tr id="input-value">
         <td><b>inputValue</b></td>
-        <td><i>''</i></td>
-        <td>Input field initial value.</td>
+        <td><i>''<br>Promise</i></td>
```
Comment (limonte) [inline index.html]: This is the default value, it's still the empty string, no need in mentioning Promise here.

File: index.html [hunk 2]
```diff
@@ -751,8 +783,8 @@ <h3 id="configuration">Configuration</h3>
...
+        <td>Input field initial value. If the input type is <strong>text</strong>, <strong>email</strong>, <strong>number</strong>, <strong>tel</strong> or <strong>textarea</strong> a Promise is accepted as inputValue.</td>
```
Comment (limonte) [inline index.html]: > a Promise is accepted as inputValue

a Promise **can be** accepted as inputValue. [See the example](#inputValue-promise).

**Comment by gverni** [comment]
Thanks for the review @limonte. I update the PR. 

Regarding the default value, i've seen that the default value for the inputOptions is "{} Map Promise": 

https://github.com/sweetalert2/sweetalert2.github.io/blob/5a60779e4685b7e89f64d3db8b6524bb45fca527/index.html#L759

Should that be updated to {} only? Or maybe i didn't understand your comment on the default value :-)

**Comment by limonte** [comment]
There's now the demo example and docs about `inputValue` as a promise, thus I'm closing this PR.

**Pull request closed without merging**